sev/Makefile.toml

```diff
@@ -0,0 +1,3 @@
+[tasks.test]
+command = "cargo"
+args = ["test", "--features=openssl"]
```

Comment:
Does `cargo make` offer us the ability to run tests on all combinations of feature flags?

Reply:
I think that would be a very powerful upstream feature. But, again, I'm happy to take that as `debt`.